Components/CreateTrip/CreateTrip.js

```diff
@@ -1,39 +1,145 @@
 import React from 'react';
-import { StyleSheet, Text, View, TextInput, TouchableOpacity } from 'react-native';
+import { Image, StyleSheet, Text, View, TextInput, TouchableOpacity } from 'react-native';
 
-export const CreateTrip = () => {
+export const CreateTrip = ({ route }) => {
   return (
     <View style={styles.container}>
-      <Text>Trip name:</Text>
-      <TextInput style={styles.input}
-        placeholder='e.g. Spring break'
-      />
-      <Text>Starting Location:</Text>
-      <TextInput style={styles.input}
-        placeholder='e.g. trip origin'
-      />
-      <Text>Destination:</Text>
-      <TextInput style={styles.input}
-        placeholder='e.g. trip destination'
-      />
-      <TouchableOpacity>
+      <View style={styles.header}>
+        <Image
+          style={styles.image}
+          source={{uri: 'https://img.icons8.com/officel/40/000000/globe.png'}}
+        />
+        <Text style={{fontSize: 20, marginLeft: 33}}>
+          New Trip
+        </Text>
+        <Text style={{color: '#96CDEC', marginRight: 1}}>
+          placehold
```

Comment:
Cool this is the line under New Trip right? I think we can make the header say New Trip if we think that would look better and we can even try to throw the logo up there too.

Reply:
Probably best to strip out the redundant headers. I'll make a styling branch and include that as an issue for it